backend/travel_diary/models.py

```diff
@@ -7,10 +7,17 @@
 
 class Travel(models.Model):
     name = models.CharField(
-        'Наименование путешествия',
-        max_length=200,
+        'Название путешествия',
+        max_length=100,
         unique=True,
-        help_text='Укажите наименование путешествия'
+        help_text='Укажите название путешествия'
+    )
+    description = models.CharField(

```

Comment:
Может быть models.TextField?

Reply:
If you specify a max_length attribute, it will be reflected in the [Textarea](https://docs.djangoproject.com/en/4.0/ref/forms/widgets/#django.forms.Textarea) widget of the auto-generated form field. However it is not enforced at the model or database level. Use a [CharField](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.CharField) for that.